conftest.py

```diff
@@ -1,5 +1,8 @@
+import os
 import pytest
 import torch
+import torchtext
+import torchvision
```

Comment:
Is this the intent to throw an exception if torchtext or torchvision is not found?

May be it's better to move those import inside pytest_benchmak_update_machine_info?
```suggestion
```